helpers/jobsQueue.js

```diff
@@ -5,18 +5,24 @@ var jobsQueue = {
 	jobs: {},
 
 	register: function (name, job, time) {
-		if (this.jobs[name]) {
+		// Check if job is already registered - we check only if property exists, because value can be undefined
```

Comment:
@MaciejBaj It's because the way how is assigned (line 24), here is example: https://runkit.com/4miners/597237d46c020c00129785c9
Should I switch to alternate approach?